packages/jest-environment/src/index.ts

```diff
@@ -349,6 +349,14 @@ export interface Jest {
    * It is recommended to use `jest.mock()` instead. The `jest.mock()` API's second
    * argument is a module factory instead of the expected exported module object.
    */
+  /**
```

Comment:
The above TSDoc belongs to 'setMock'. Would be good to move it down (;

Reply:
Thanks! Fixed